tools/src/h5repack/h5repack.c

```diff
@@ -633,7 +663,7 @@ check_options(pack_opt_t *options)
      *-------------------------------------------------------------------------
      */
 
-    if (options->verbose && have_request(options)) {
+    if (options->verbose > 0 && have_request(options)) {
```

Comment:
Why this change's neccessary?

Reply:
Consistency - since I added value 2 to be timing I figured it would be better to check the value the same way for all verbose vars.

Reply:
Got it.